README.md

```diff
@@ -91,3 +91,39 @@ Here is a breakdown of the fields in the ```wedding_invite.yml``` file:
 | config | A dictionary of environment variables that will be used to build the service.                    |
 | build_args | A dictionary of build arguments that will be used to build the service.                          |
 
+# Using wedpy locally
+Prerequisites:
+
+1. Open the target repository and have docker desktop running in the background.
```

Comment:
I would add a Hyperlink to Docker Desktop website so you can download the right version.